{
  "Header": {
    "Original": "CVE-2019-14889 (A flaw was found with the libssh API function ssh_scp_new() in version ...)",
    "Line": 280778,
    "ID": "CVE-2019-14889",
    "Description": "(A flaw was found with the libssh API function ssh_scp_new() in version ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-3437-1 DLA-2038-1}",
      "Line": 280779,
      "Type": "xref",
      "Bugs": [
        "DLA-3437-1",
        "DLA-2038-1"
      ]
    },
    {
      "Original": "- libssh 0.9.3-1 (bug #946548)",
      "Line": 280780,
      "Type": "package",
      "Package": "libssh",
      "Kind": "fixed",
      "Version": "0.9.3-1",
      "BugNo": 946548
    },
    {
      "Original": "[stretch] - libssh \u003cno-dsa\u003e (Minor issue)",
      "Line": 280781,
      "Type": "package",
      "Release": "stretch",
      "Package": "libssh",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "NOTE: https://www.libssh.org/security/advisories/CVE-2019-14889.txt",
      "Line": 280782,
      "Type": "NOTE",
      "Description": "https://www.libssh.org/security/advisories/CVE-2019-14889.txt"
    },
    {
      "Original": "NOTE: https://bugs.libssh.org/T181",
      "Line": 280783,
      "Type": "NOTE",
      "Description": "https://bugs.libssh.org/T181"
    },
    {
      "Original": "NOTE: The fix in libssh makes an update in x2goclient necessary, cf:",
      "Line": 280784,
      "Type": "NOTE",
      "Description": "The fix in libssh makes an update in x2goclient necessary, cf:"
    },
    {
      "Original": "NOTE: https://bugs.debian.org/947129",
      "Line": 280785,
      "Type": "NOTE",
      "Description": "https://bugs.debian.org/947129"
    },
    {
      "Original": "NOTE: https://code.x2go.org/gitweb?p=x2goclient.git;a=commitdiff;h=ce559d163a943737fe4160f7233925df2eee1f9a",
      "Line": 280786,
      "Type": "NOTE",
      "Description": "https://code.x2go.org/gitweb?p=x2goclient.git;a=commitdiff;h=ce559d163a943737fe4160f7233925df2eee1f9a"
    }
  ]
}